src/iso.h

```diff
@@ -0,0 +1,103 @@
+/*
+Copyright (C) 2016 The University of Notre Dame
+This software is distributed under the GNU General Public License.
+See the file LICENSE for details.
+*/
+
+#ifndef ISO_H_
+#define ISO_H_
+
+#include "kerneltypes.h"
```

Comment:
If we are including kernel types, do we want to use the bool type that is provided in there for function returns (where appropriate) instead of an int?